### DISCUSSION TITLE
Content media field values are storing in different formats on J3 & J4.

### DISCUSSION THREAD

**Discussion by trananhmanh89** [joomla/joomla-cms#37886] [General]
Hi,
Correct if i'm wrong.
In J3, media field is storing by image url.
![image](https://user-images.githubusercontent.com/20571336/170173043-b86a02b4-88fc-4938-a821-373828fc23e5.png)

But j4, it's a JSON object.
![image](https://user-images.githubusercontent.com/20571336/170173248-55f3b798-72db-486f-809c-b8ff0654e218.png)

...
This issue breaks my extension on j4. 
A bit angry 😡

**Comment by joomdonation** [comment]
It is one of the backward incompatible change documented here https://docs.joomla.org/Potential_backward_compatibility_issues_in_Joomla_4#Media_Field .
